src/textual/screen.py

```diff
@@ -101,6 +101,12 @@ class Screen(Generic[ScreenResultType], Widget):
     }
     """
 
+    auto_focus: str | None = "*"
```

Comment:
Let's make this an instance var, which should be the default unless there is good rationale for it being a classvar.

Reply:
Mostly because it's a pain to set it as an instance attribute only for people creating their own screens:

```py
class FormScreen(Screen):
    auto_focus = "Input"

    ...
```

vs 

```py
class FormScreen(Screen):
    def __init__(
        self,
        name: str | None = None,
        id: str | None = None,
        classes: str | None = None,
    ) -> None:
        self.auto_focus = "Input"
        super().__init__(name=name, id=id, classes=classes)

    ...
```

Reply:
Classvar it is.